express/parsers/apps/aiida/formats/zip.py

```diff
@@ -0,0 +1,84 @@
+import esse
+
+from express.parsers.formats.zip import BaseZipParser
+from express.parsers.formats.jsonfile import BaseJSONParser
+
+
+SUPPORTED_ARCHIVE_VERSION = '0.9'
+SUPPORTED_AIIDA_VERSION = '1.4.2'
+
+ES = esse.ESSE()
+SCHEMA_MATERIAL = ES.get_schema_by_id('material')
+
+
+class AiidaZipParser(BaseZipParser):
+
+    def __init__(self, *args, **kwargs):
+        self.json_parser = BaseJSONParser
+        super(AiidaZipParser, self).__init__(*args, **kwargs)
+
+    def structures(self):
+        structures = []
```

Comment:
Although the best documentation to code is code itself, and this is well written, it might still be helpful to add a dosctring to briefly explain what is happening in the function.